js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -212,9 +213,30 @@ const DropDownEditor = TextBox.inherit({
         this._initVisibilityActions();
         this._initPopupInitializedAction();
         this._updatePopupPosition(this.option('rtlEnabled'));
+        this._mergeDropDownOptions();
+
         this._options.cache('dropDownOptions', this.option('dropDownOptions'));
     },
 
+    _mergeDropDownOptions: function() {
+        const options = this.option('dropDownOptions');
+        const defaultConfig = this._popupConfig();
+        const wrapperClasses = this._mergeWrapperClasses(defaultConfig, options);
+
+        if(options.wrapperAttr?.class) {
+            options.wrapperAttr.class = wrapperClasses;
```

Comment:
I think it's not a good idea to mutate public option's value

Reply:
We can't add this option by another way.